upup/models/cloudup/resources/addons/networking.projectcalico.org.canal/k8s-1.15.yaml.template

```diff
@@ -422,31 +456,24 @@ subjects:
   name: canal
   namespace: kube-system
 ---
-# Bind the Calico ClusterRole to the canal ServiceAccount.
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRoleBinding
 metadata:
   name: canal-calico
+  labels:
+    role.kubernetes.io/networking: "1"
 roleRef:
   apiGroup: rbac.authorization.k8s.io
   kind: ClusterRole
-  name: calico
+  name: calico-node
```

Comment:
Due to the change in role name upstream, this becomes a breaking change for users upgrading as role references in bindings cannot be changed.

Reply:
ACK. Will change back this part.